helm/prometheus-kafka-adapter/templates/service.yaml

```diff
@@ -14,6 +14,7 @@ spec:
   ports:
     - port: {{ .Values.service.port }}
       targetPort: http
+      nodePort: {{ .Values.service.nodeport }}
```

Comment:
Should this line be conditioned to the existance of .Values.service.nodeport?
I think it's undefined by default.

Reply:
yes, done and checked